common/build.gradle

```diff
@@ -274,6 +274,14 @@ afterEvaluate {
                     password project.vstsPassword
                 }
             }
+            maven {
+                name "vsts-maven-android"
+                url 'https://identitydivision.pkgs.visualstudio.com/IDDP/_packaging/Android/maven/v1'
+                credentials {
+                    username project.vstsUsername
+                    password project.vstsPassword
+                }
+            }
```

Comment:
Do we need to do this? Given that we already did in project level build.gradle file

Reply:
Nope.  I just removed it... as it was breaking the VSTS pipeline.

Reply:
Okay that was for a different feed...resolving